site/docs/spark-queries.md

```diff
@@ -220,27 +218,140 @@ SELECT * FROM prod.db.table.files
 
 ### Manifests
 
-To show a table's file manifests and each file's metadata, run:
+To show a table's current file manifests and each file's metadata, run:
```

Comment:
I think there was a comment from @rdblue about not put run here and in the other ones:  https://github.com/apache/iceberg/pull/3482#discussion_r744297228

Reply:
oh，sorry，Refer to other documents，
"To show a table's current file manifests and each file's metadata:   " I think will be better